src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --main-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--main-color);
+}
+
+body,
+.header,
+.header-logo,
+.menu,
+.menu-item,
+.menu-link,
+::after {
+  box-sizing: border-box;
+}
```

Comment:
You should avoid using the '*' selector as it impacts performance. Instead, set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser.